ansible/group_vars/all/vars.yml

```diff
@@ -9,6 +9,7 @@ passwd:
   afflower: {login: afflower, id: 2005, comment: Airflow Celery Flower}
   plpsql:   {login: plpsql,   id: 2100, comment: Pipeline Postgresql}
   plshovel: {login: plshovel, id: 2101, comment: Pipeline Shovel}
+  oomsmweb: {login: oomsmweb, id: 2300, comment: OONI Measurements WebAPI}
```

Comment:
Yep, by piwik. Okay, I'll renumber `oomsmweb`.
I suggest following way to take further base UIDs for group of constellation of services to prevent race: `echo $(( 2000 + 50*($RANDOM % 160) ))`.